account_invoice_report_hide_line/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import account_invoice
```

Comment:
@bodedra Missing newline.

Can you also change the copyright header to:
```
# Copyright (C) 2019 Open Source Integrators
```

Thanks!